src/mvBasicWidgets.cpp

```diff
@@ -6304,7 +6336,11 @@ DearPyGui::draw_separator(ImDrawList* drawlist, mvAppItem& item)
 {
 	if (!item.config.show)
 		return;
-	ImGui::Separator();
+	if (!item.info.internalLabel.empty()) {
```

Comment:
Unfortunately this condition doesn't work as expected: even if the user does not specify the label, or specifies it as `label=""`, `internalLabel` is not empty. It appends the item's UUID as a `###` suffix, like `"my_label###32"` or `"###28"`. This makes `add_separator` actually go into the `SeparatorText` branch, and the separator looks different from what it used to be:
1. Its thickness is 3 pixels by default (was 1 pixel in the old version).
2. It adds extra padding, like a line of text does, even though the separator itself doesn't output any text.

With the current code, the only way to get the old good thin separator is
```py
dpg.add_separator(use_internal_label=False, label="")
```
(yes, **both** arguments are necessary).

To make it less disruptive, I'd suggest to test for `item.config.specifiedLabel.empty()` instead. This will make the plain `dpg.add_separator()` call work exactly like the old version - no need for fancy `use_internal_label` tricks.